File: barbeariainclusao/barbeariainclusao/barbeariainclusao/barbeariainclusao/barbeariainclusao/src/main/java/br/cefetmg/barbeariainclusao/model/Avaliacao.java
package br.cefetmg.barbeariainclusao.model;

import lombok.*;

@Data
@NoArgsConstructor
@AllArgsConstructor

public class Avaliacao {

    private Long idAvaliacao;
    private int nota;
    private String comentario;
    private long idAtendimentoAvaliacao;

}

File: BarbeariaEInclusao/src/app/model/avaliacao.ts
export class Avaliacao {
    idAvaliacao: number;
    nota: number;
    comentario: string;
    idAtendimentoAvaliacao: number;

    constructor() {
        this.idAvaliacao = 0;
        this.nota = 0;
        this.comentario = '';
        this.idAtendimentoAvaliacao = 0;
    }
}